src/libraries/System.ComponentModel.TypeConverter/tests/TrimmingTests/System.ComponentModel.TypeConverter.TrimmingTests.proj

```diff
@@ -1,5 +1,12 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
-
+  <ItemGroup>
+    <TestConsoleAppSourceFiles Include="DesignTimeLicenseContextSerializer.cs">
+      <!-- Setting the Trimming feature switch to make sure that the Resources get trimmed by the linker
```

Comment:
Comment is wrong (copy-pasted from a different test). That said, please look at my comment here [https://github.com/dotnet/runtime/pull/48527#discussion_r588603078](https://github.com/dotnet/runtime/pull/48527#discussion_r588603078) as I don't think that you should be adding a trimming test at all for your case. It is fine to keep this locally while developing so that you can validate that BF stuff gets trimmed correctly, but the trimming test would not be really useful as it is not really testing correctness here.